src/inference/src/cache_manager.hpp

```diff
@@ -137,16 +138,15 @@ class FileStorageCacheManager final : public ICacheManager {
         ScopedLocale plocal_C(LC_ALL, "C");
         auto blob_file_name = getBlobFile(id);
         if (ov::util::file_exists(blob_file_name)) {
+            Tensor compiled_blob;
             if (enable_mmap) {
```

Comment:
I think the PR https://github.com/openvinotoolkit/openvino/pull/29449 will help to unify these code branches to a single one.